java/libphonenumber/test/com/google/i18n/phonenumbers/RegionCode.java

```diff
@@ -14,51 +14,49 @@
  * limitations under the License.
  */
 
+/* This file is automatically generated by {@link BuildMetadataProtoFromXml}.
+ * Please don't modify it directly.
+ */
+
 package com.google.i18n.phonenumbers;
 
-/**
- * Class containing string constants of region codes for easier testing.
- */
 final class RegionCode {
+  // Class containing string constants of region codes for easier testing.
```

Comment:
Sorry, I just now noticed that this is not a Javadoc comment anymore. Could you also change the `ClassWriter` to accept a Javadoc on top of the class? Also consider a Builder pattern for the `ClassWriter`, as you would have now different combinations of constructor parameters. You could use [AutoValue.Builder](https://github.com/google/auto/blob/main/value/userguide/builders.md) to have it autogenerated.